semgrep-core/src/api/AST_generic_v1.atd

```diff
@@ -466,6 +466,9 @@ type argument = [
   | Arg of expr (* can be Call (IdSpecial Spread, Id foo) *)
   (* keyword argument *)
   | ArgKwd of (ident * expr)
+  (* optional keyword argument - can occur in target code without being
```

Comment:
Actually in theory we should not change those _v1.atd because it's supposed to be a fixed AST.
Nobody uses this _v1 for now, so no big deal, but the spirit of those _vxxx.atd is to be fixed. 
See the toplevel comment in this file: "DO NOT MODIFY THIS FILE ..."

Reply:
oh yes, I wanted to ask about that. What we should we do when we run into a new construct? The simplest would be to raise an exception. What do you think?

Reply:
e.g. when exporting the generic AST to json we'd have this:
```ocaml
| ArgKwdOptional _ ->
    raise (Unsupported_construct "ArgKwdOptional")
```

Reply:
Right now I raise/failwith because anyway this code is not really used, but in theory we should try to convert to something similar (here we could convert to an ArgKwd, better than nothing)